test/pubchem/github.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { expect } from 'chai';
+import GithubService from '../../src/services/github.service';
+
+describe('GitHub API with Octokit testing', () => {
+	it('should return with not error for contributors', async () => {
+		const githubService = new GithubService();
+		const contributors = await githubService.getContributors(
```

Comment:
This is integration test. What we need is unit testing, ensure the functionality of the service working as expected. Therefore you need to mock / stub your function, in other words, have the response and _assert_ that. 

For example a good test case would be like: Can service merge the repos and group by user?

Reply:
Maybe in the future we can have integration test though. For now, it's okay not to have.

Reply:
As an example or get the idea, you may check PubChem Import Script task's tests #16